content/about/working-groups.fr.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Working Groups
```

Comment:
Just a quick typo: title should be `working-groups` and not `Working Groups`, I know it's minimal, but it's just to guarantee, your file metadata matches the english one!